examples/02_meta-analyses/10_plot_cbmr.py

```diff
@@ -0,0 +1,362 @@
+dset.annotations["sample_sizes"] = [dset.metadata.sample_sizes[i][0] for i in range(n_rows)]
+dset.annotations["avg_age"] = np.arange(n_rows)
+# set up categorical moderators: schizophrenia_subtype (as not enough data to be interpreted
+# as groups)
+dset.annotations["schizophrenia_subtype"] = ["type1", "type2", "type3", "type4", "type5"] * int(
+    n_rows / 5
+)
+dset.annotations["schizophrenia_subtype"] = (
+    dset.annotations["schizophrenia_subtype"].sample(frac=1).reset_index(drop=True)
+)  # random shuffle drug_status column
+
+###############################################################################
+# Estimation of group-specific spatial intensity functions
+# -----------------------------------------------------------------------------
+# Unlike kernel-based CBMR methods (e.g. ALE, MKDA and SDM), CBMR provides a
+# generative regression model that estimates a smooth intensity function and
+# can have study-level moderators. It's developed with a spatial model to
+# induce a smooth response and model the entire image jointly, and fitted with
+# different variants of statistical distributions (Poisson, Negative Binomial
+# (NB) or Clustered NB model) to find the most accurate but parsimonious model.
+#
+# CBMR framework can generate estimation of group-specific spatial internsity
+# functions for multiple groups simultaneously, with different group-specific
+# spatial regression coefficients.
+#
+# CBMR framework can also consider the effects of study-level moderators
+# (e.g. sample size, year of publication) by estimating regression coefficients
+# of moderators (shared by all groups). Note that moderators can only have global
+# effects instead of localized effects within CBMR framework. In the scenario
+# that there're multiple subgroups within a group, while one or more of them don't
+# have enough number of studies to be inferred as a separate group, CBMR can
+# interpret them as categorical study-level moderators.
+from nimare.meta.cbmr import CBMREstimator
+
+dset = StandardizeField(fields=["sample_sizes", "avg_age"]).transform(dset)
+
+cbmr = CBMREstimator(
+    group_categories=["diagnosis", "drug_status"],
+    moderators=[
+        "standardized_sample_sizes",
+        "standardized_avg_age",
+        "schizophrenia_subtype:reference=type1",
+    ],
+    spline_spacing=100,  # a reasonable choice is 10, 100 is for speed
+    model=models.PoissonEstimator,
+    penalty=False,
+    lr=1e-1,
+    tol=1e3,
+    device="cpu",
+)
+results = cbmr.fit(dataset=dset)
+plot_stat_map(
+    results.get_map("spatialIntensity_group-SchizophreniaYes"),
+    cut_coords=[0, 0, -8],
+    draw_cross=False,
+    cmap="RdBu_r",
+    title="Schizophrenia with drug treatment",
+    threshold=1e-4,
+)
+plot_stat_map(
+    results.get_map("spatialIntensity_group-SchizophreniaNo"),
+    cut_coords=[0, 0, -8],
+    draw_cross=False,
+    cmap="RdBu_r",
+    title="Schizophrenia without drug treatment",
+    threshold=1e-4,
```

Comment:
Why this threshold?

Reply:
When plotting the maps of group-wise spatial intensity estimation (not z-score maps), I choose the threshold 1e-4 so that there're some activation regions (voxelwise estimated intensity above the average) highlighted, while not majority of brain region are highlighted. But this threshold depends on the size of dataset.